handling_data.rb

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+# module
+module CheckApplicability
+  def loan_applicability?(population, gdp, states, state)
+    if population > 200_000 && gdp >= 20_000 && states >= 5 && state == 'developing'
+      return true
+    end
+
+    false
```

Comment:
By refactoring I meant: The if else can be avoided as it can be replaced by:
```ruby
def loan_applicability?(population, gdp, states, state)
  population > 200_000 && gdp >= 20_000 && states >= 5 && state == 'developing'
end
```
It is because statement is conditional and will always evaluate to true or false and since it is last statement we dont have to return